superset-frontend/src/explore/components/controls/AdhocFilterControl.jsx

```diff
@@ -125,9 +128,10 @@ export default class AdhocFilterControl extends React.Component {
                 );
               }
             }
-          },
-          // no error handler, in case of error do not show partition option
-        );
+          })
+          .catch(error => {
+            console.log('fetch extra_table_metadata:', error.statusText);
```

Comment:
can you add an eslint disable comment here to prevent the warning?

and maybe this should be a `console.error`?

Reply:
fixed. console.error is better.